subprojects/gst-plugins-bad/sys/msdk/gstmsdk.c

```diff
@@ -67,6 +67,11 @@
 #endif
 #include "gstmsdkvpp.h"
 
+#if (MFX_VERSION >= 2000)
```

Comment:
@liuyinhangx, could you explain why we need #if here?

Reply:
In the previously implemented version, MFX_CODEC_VP8 and MFX_CODEC_JPEG are used in the VPL path but not in the MSDK path, so this conditional compilation is added. In the current version, MFX_CODEC_VP8 and MFX_CODEC_JPEG are used in both VPL and MSDK path. I forgot to delete this conditional compilation, I will correct it later. Thanks.

Reply:
Fixed, removed the conditional compilation.